### DISCUSSION TITLE
Need help with schematic view...

### DISCUSSION THREAD

**Discussion by kpgrafix** [FiercestT/EasyEdaThemes#14] [Q&A] [answered]
I like the dark theme especially working at night/low light environment BUT for the schematic diagram view I'd like to see the original colors of the symbols, pins, and wires. We use a lot of custom symbols with color designations etc. Im no coder but I can follow if anybody can show me how... thanks!

So basically dark theme everywhere except the schematic/sheet view window, if this is easier to accomplish

**Comment by FiercestT** [comment, marked as answer]
Try commenting out the contents of this function in `main.js`, then reimport the extension with the edited file. It should prevent the editor from applying the custom schematic theme. Then, in EasyEda [View] -> [Theme] -> [Original Theme]

https://github.com/FiercestT/EasyEdaThemes/blob/aba3259697438fbb0610ba507ccd7babebed11b4/extension/main.js#L80-L89

to this

```js
 ///Sets the schematic editor theme 
 function setEditorTheme(themeData) { /*
     api('doCommand', 'applyThemeUserDefine') 
     api('editorCall', { 
        cmd: 'setColors', 
        args: [{ 
            activeTheme: 'user-define', 
            color: MAPPING(themeData) 
     }]}) */
 } 
```
